src/layout/layout.js

```diff
@@ -370,11 +466,16 @@
       if (this.drawer_) {
         var drawerButton = this.element_.querySelector('.' +
           this.CssClasses_.DRAWER_BTN);
+        var drawerNav = this.drawer_.querySelector('.mdl-navigation');
+        if (drawerNav) {
+          drawerNav.setAttribute('role', 'menu');
```

Comment:
Why add this attribute internally? Aren't developers able to add the role in their own code if they want it?